buildscripts/ci/windows/package.bat

```diff
@@ -239,7 +222,7 @@ COPY %FILEPATH% %ARTIFACTS_DIR%\%ARTIFACT_NAME% /Y
 SET ARTIFACT_PATH=%ARTIFACTS_DIR%\%ARTIFACT_NAME%
 
 IF %DO_SIGN% == ON (
-    CALL %SIGN% --secret %SIGN_CERTIFICATE_ENCRYPT_SECRET% --pass %SIGN_CERTIFICATE_PASSWORD% --name %ARTIFACT_NAME% --file %ARTIFACT_PATH% || exit \b 1
+    CALL %SIGN% --secret %SIGN_SECRET% --key %SIGN_KEY% --file %ARTIFACT_PATH% || exit \b 1
```

Comment:
Isn't it `/b` instead of `\b`? (it was already like that apparently in multiple places; probably a copy/pasted mistake)

Reply:
fixed